cluster-scope/overlays/prod/osc/osc-cl2/kustomization.yaml

```diff
@@ -56,6 +56,7 @@ resources:
 - ../../../../bundles/openshift-pipelines
 - ../../../../bundles/pachyderm-operator
 - ../../../../bundles/reloader
+- ../../../../bundles/kafka-operator
```

Comment:
Not certain but I believe the subscription to the  Strimzi operator should also  be added to the overlay for osc-cl2, [as it is  done for smaug](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml#L113)